prime-dotnet-webapi/ViewModels/Enrollee/_EnrolleeMappingProfile.cs

```diff
@@ -37,7 +38,10 @@ public EnrolleeMappingProfile()
             CreateMap<EnrolleeDTO, EnrolleeViewModel>();
 
             CreateMap<EnrolleeNote, EnrolleeNoteViewModel>();
-            CreateMap<EnrolleeAbsence, EnrolleeAbsenceViewModel>();
+
+            CreateMap<EnrolleeAbsence, EnrolleeAbsenceViewModel>()
+                .ForMember(dest => dest.StartTimestamp, opt => opt.MapFrom(src => src.StartTimestamp.ToLocalTime()))
+                .ForMember(dest => dest.EndTimestamp, opt => opt.MapFrom(src => src.EndTimestamp.HasValue ? (DateTime?)src.EndTimestamp.Value.ToLocalTime() : null));
```

Comment:
I think the slightly better way to do this is using a precondition, like we do for agreements:
![image](https://user-images.githubusercontent.com/39168456/148279518-da2077f2-139d-4e5a-b4ef-ff965f04ce24.png)